src/stories/Foundation/Icon/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import * as icons from './icons';
+import { StandardColors } from '../../../shared/types';
+
+export interface IconProps {
+  className?: string;
+  color?: StandardColors;
+  size?: 24 | 48;
+  name: keyof typeof icons;
+}
+
+// name should accept the name of IconProps
+const getIcon = (name: IconProps['name'], size: IconProps['size']) => {
```

Comment:
Look at this functional programming placing the function outside the main function 🥇

Reply:
Learning faster than an AI B)